projects/sandbox/plots/plots/sensitive_volume.py

```diff
@@ -98,6 +98,13 @@ def main(
     y *= v0
     err *= v0
 
+    # collect a VT measurement for each mass combo
```

Comment:
Being picky here, but I think we should avoid using "VT" here and in `vt_vs_mass` because in the pipeline comparison section, a distinction is made between V and VT. Maybe swap to "SV"?